templates/rhpam73-authoring.yaml

```diff
@@ -188,17 +188,17 @@ parameters:
   required: true
 ## OpenShift Enhancement BEGIN
 - displayName: Prefer KIE Server OpenShift Service [Tech Preview]
-  description: Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
+  description: If OpenShift Integration of the Business Central is turned on, set this parameter to true to enables connection to KIE Server via an OpenShift internal Service endpoint. Otherwise the Business Central uses a Route to connect to KIE Server. (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
```

Comment:
I do not believe "Otherwise the Business Central uses a Route to connect to KIE Server." is correct. @rhtevan , please suggest a better description?

Reply:
Route removed from description for now; not resolving in PR to allow for discussion between David and Evan.